upgrade/helpers/tasks.py

```diff
@@ -217,32 +246,57 @@ def _sync_rh_repos_to_satellite(org):
     scl_reposet = entities.RepositorySet(
         name=rhelcontents['rhscl']['repo'].format(os_ver=rhelver), product=scl_product
     ).search()[0]
+    logger.info("Red Hat Software Collection for repos {} is already enabled and found".
+                format(rhelcontents['rhscl']['repo'].format(os_ver=rhelver)))
     try:
         scl_reposet.enable(data={'basearch': arch, 'releasever': '7Server'})
+        logger.info("Red Hat Software collection repository enabled successfully")
     except requests.exceptions.HTTPError:
         logger.warn('Check if repository is already enabled, else retrigger')
     time.sleep(20)
     # Sync enabled Repo from cdn
     scl_repo = entities.Repository(
         name=rhelcontents['rhscl']['repofull'].format(os_ver=rhelver, arch=arch)
     ).search(query={'organization_id': org.id, 'per_page': 100})[0]
-    call_entity_method_with_timeout(entities.Repository(id=scl_repo.id).sync, timeout=2500)
+    attempt = 0
+    while attempt <= 3:
+        try:
+            call_entity_method_with_timeout(
+                entities.Repository(id=scl_repo.id).sync, timeout=2500)
+            break
+        except requests.exceptions.HTTPError as exp:
+            logger.warn("Retry{} after exception: {}".format(attempt, exp))
+            # Wait 10 seconds to reattempt the same retry option
```

Comment:
Why do you want to retry for repo sync ?

Reply:
I will create a separate nailgun issue for the same.